src/components/common/RenameNodeModal.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Form, Input, Modal } from 'antd';
+import { usePrefixedTranslation } from 'hooks';
+import { CommonNode } from 'shared/types';
+import { useStoreActions, useStoreState } from 'store';
+import { useAsyncCallback } from 'react-async-hook';
+import { Network } from 'types';
+
+interface Props {
+  network: Network;
+}
+
+const RenameNodeModal: React.FC<Props> = ({ network }) => {
```

Comment:
![image](https://github.com/jamaljsr/polar/assets/1356600/d4f5d017-f583-4cb7-b68d-c9201a91514c)

The modal looks a bit empty. I'd suggest the following updates:
- Update title to be "Rename Node <name>" so the old name is displayed even after modifying the input value
- Add a label to the input: "New Node Name"
- If the network is started, add a message at the top stating "The network will be restarted to perform this operation". You can use `<Alert type="warning" message="..." />` to make it look nice.

Reply:
> Update title to be "Rename Node " so the old name is displayed even after modifying the input value

I don't understand what you mean by this

Reply:
Oh I didnt realize Github hid the text `<name>` in my comment. I guess it thinks it's html. I was suggesting to change the title to include the current node's name. Ex: "Rename Node alice"

Reply:
ooh I get you now. Let me work on that